docs/source/description.rst

```diff
@@ -2,3 +2,17 @@ Alot is a terminal-based mail user agent for the notmuch mail system.
 It features a modular and command prompt driven interface
 to provide a full MUA experience as an alternative to the Emacs mode shipped
 with notmuch.
+What you can do whit alot:
+
+	* Read mail/thread
+	* Search mail
+	* Tag/Untag mail
+	* Filter mail	
+	* Save attachment
+	* Receive mail (using external tool as `mbsync <http://isync.sourceforge.net/mbsync.html>`_ or `offlineIMAP <http://www.offlineimap.org/>`_)
+	* Send/Reply to/Bounce/Forward mail (using external tool as `msmtp <https://marlam.de/msmtp/>`_)
+	* View attachment (using externals tools)
+
+What you can't do with alot:
+	
+	* Search in/save mail
```

Comment:
These are two different things in a single line, also have you opened an issue for the "save mail" thing? I shouldn't be that hard to do. You could probably use `:pipeto` with `cat` to make that work, and it would be trivial to implement in alot itself.